addons/l10n_fr_pos_cert/static/src/js/pos.js

```diff
@@ -52,6 +52,13 @@ patch(Order.prototype, {
         result = Boolean(result || this.pos.is_french_country());
         return result;
     },
+    _get_qr_code_data() {
+        if (this.pos.is_french_country()){
+          return false;
+        } else {
+          return this._super(...arguments);
```

Comment:
use `super._get_qr_code_data()`

Reply:
My mistake, it's not used anymore in model.js in `export_for_printing()` so there is no override to begin with.